rapier/rules-update-2020-07.md

```diff
@@ -0,0 +1,63 @@
+
+The marshal-in-charge of an activity may limit Rapier Spears on a per-activity basis as long as the limitation is widely announced.  As an example, after bringing all participants together, announce: "For the next melee, you may not use Rapier Spears."
+
+# Parry-Only Daggers
+
+All equipment with a steel or rubber blade must pass the blade requirements.  Equipment without a blade is not an offensive weapon in the context of Rapier in Atlantia.  As an example, an armored combat rattan dagger does not have a blade, and therefore can be used as a parrying device, similar to the already approved wooden dowel rod or rolling pin.
+
+# Knee-Walking
+
+Knee-Walking is allowed by society policy at all times.  The marshal-in-charge of an activity may limit knee-walking on a per-activity basis as long as the limitation is widely announced.  As an example, after bringing all participants together, announce: "For the next melee, you may only knee walk to engagement."
+
+# C&T Melee
+
+The new society rules allow for melees using C&T combat.  All participants in C&T melee must hold an active C&T authorization and must be actively marshaled by someone approved to marshal C&T combat in Atlantia.  Participants with a Rapier Spear authorization without a C&T authorization may participate in C&T melees using only a Rapier Spear.
+
+All those marshaling C&T melee should record and submit detailed feedback in the Event Report as to the efficacy, safety, and any issues raised during C&T melee activities.
+
+# C&T blows to the legs
+
+In C&T, blows to the leg are no longer considered "fighting-ending".  Participants that receive a blow to the leg must follow standard rapier guidelines for blows to the leg.  As with rapier, participants that do not wish to continue after receiving a blow to the leg may choose to take any blow as a "killing" blow.
+
+# Updated guidance for armor coverage
+
+The [new society handbook](https://www.sca.org/wp-content/uploads/2020/04/FencingMarshalsHandbookApril2020.pdf) includes images that explicitly defines the minimum armor coverage required for torso armor, gorget, and mask coverage on pages 19, 20, and 21 respectively.  All participants must meet these standards.
+
+# Thrown Cloaks
+
+Cloaks may be thrown towards an opponent, but may specifically not throw cloaks over an opponent's face or with the intent to trip an opponent.  
+
+# New "Front 180" rules
+
+The [new society handbook](https://www.sca.org/wp-content/uploads/2020/04/FencingMarshalsHandbookApril2020.pdf) includes specific language that defines where participants can strike their opponents:
+
+This includes:
+* 5.3.2. Combatants may strike any single opponent they can safely reach with any legal blow if they are within a 180 degree arc of the opponent’s front as defined by the opponent’s shoulders, and at an angle they can be reasonably seen by the opponent.

```

Comment:
So in a melee, if I am in combat with a single opponent my outside lines are unassailable if I take a narrow stance with my rear shoulder fully back?

If I am a line of 1 v a line of 1 to negate this issue, is 5.3.2 actually real?

Reply:
Would hips be a better reference point so that we can protect people from blows to the back.

Reply:
Hips creates the same problems. The real solution is that, since what we really care about are hits to the spine/base of skull, we move to a front 270 degrees and cue the front off the spine itself. Maybe someday.

Reply:
To Dante's original comment, yes Society's language is.... lacking.  However, it doesn't say that the blow has to be in that front 180, only that you are.  And that means any part of that front 180, even if it's the last degree.

Reply:
If I'm in a melee and someone attacks me, does a scanso della vita count as illegally maneuvering to put my back toward my opponent, and in either case, am I unable to be struck during this technique?

Reply:
From what I can gather, the front shoulder is still pointing towards an opponent during a scanso della vita keeping at least 1 degree towards them, so they can hit you anywhere they can reach you, including your butt, back, back of your head, arm, wrist, front knee, back of your back leg, etc.

Reply:
The point of the front shoulder would project a line with the opponent outside the front 180 by necessity. I don't think you can bring your torso off their line of attack correctly without having that happen.

Reply:
This will be a continued discussion as we work through this in practice.  Leaving as is.